nixos/modules/services/network-filesystems/dave.nix

```diff
@@ -0,0 +1,159 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dave;
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  options.services.dave = {
+    enable = mkEnableOption "dave WebDAV server";
+
+    user = mkOption {
+      type = types.str;
+      default = "dave-webdav";
+      description = ''
+        User account under which dave runs.
+
+        <note><para>
+          If left as the default value this user will automatically be created
+          on system activation, otherwise you are responsible for
+          ensuring the user exists before the dave service starts.
+        </para></note>
+      '';
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "dave-webdav";
+      description = ''
+        Group under which dave runs.
+
+        <note><para>
+          If left as the default value this user will automatically be created
+          on system activation, otherwise you are responsible for
+          ensuring the user exists before the dave service starts.
+        </para></note>
+      '';
+    };
+
+    openFirewall = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
Sorry missed this originally. We have an official policy to set the default value to `false` for **every** service except `openssh`.

Reply:
No problem. FYI the snapserver, avahi, and hadoop modules  violate this policy. I'm about to head to bed, but I can open an issue about it tomorrow.

Reply:
#149889